src/util/time.h

```diff
@@ -21,15 +21,20 @@ using SteadyMicroseconds = std::chrono::time_point<std::chrono::steady_clock, st
 void UninterruptibleSleep(const std::chrono::microseconds& n);
 
 /**
- * Helper to count the seconds of a duration.
+ * Helper to count the seconds of a duration/time_point.
  *
- * All durations should be using std::chrono and calling this should generally
+ * All durations/time_points should be using std::chrono and calling this should generally
  * be avoided in code. Though, it is still preferred to an inline t.count() to
  * protect against a reliance on the exact type of t.
  *
- * This helper is used to convert durations before passing them over an
+ * This helper is used to convert durations/time_points before passing them over an
  * interface that doesn't support std::chrono (e.g. RPC, debug log, or the GUI)
  */
+template <typename Clock>
+constexpr int64_t CountSeconds(std::chrono::time_point<Clock, std::chrono::seconds> t)
+{
+    return t.time_since_epoch().count();
+}
 constexpr int64_t count_seconds(std::chrono::seconds t) { return t.count(); }
```

Comment:
It's confusing to me to have both `count_seconds` and `CountSeconds`.

Reply:
Ah, I forgot that function overloading exists. Thanks, fixed.